Formula/g/groff.rb

```diff
@@ -36,7 +37,13 @@ class Groff < Formula
   end
 
   def install
-    system "./configure", "--prefix=#{prefix}", "--without-x", "--with-uchardet"
+    # Local config needs to survive upgrades
+    inreplace "Makefile.in", "localfontdir = $(dataprogramdir)",
+              "localfontdir = @sysconfdir@/groff"
+    inreplace "Makefile.in", "localtmacdir = $(dataprogramdir)",
+              "localtmacdir = @sysconfdir@/groff"
+    system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{etc}",
+           "--without-x", "--with-uchardet"
```

Comment:
```suggestion
    system "./configure", "--sysconfdir=#{etc}", "--without-x", "--with-uchardet", *std_configure_args
```

Reply:
@SMillerDev : I tried using the std_configure_args earlier, but one of the standard args isn't recognized...

Reply:
@SMillerDev: actually --disable-debug, although not recognized, only issues a WARNING -- I'll add it.